app/styles/components/map.scss

```diff
@@ -0,0 +1,10 @@
+.map {
+  height: 100%;
+  width: 100%;
```

Comment:
I think you dont need height/width if you are setting position top, bottom, left, right.